src/main.py

```diff
@@ -12,85 +12,163 @@
 from selenium.common.exceptions import NoSuchElementException
 from webdriver_manager.chrome import ChromeDriverManager
 from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
 
 # Load in the credentials from .env
 load_dotenv()
 
-# Get the Chromium web driver
-driver = webdriver.Chrome(ChromeDriverManager().install())

```

Comment:
Why was this line removed?

This line of code creates the WebDriver itself, using the ChromeDriverManager library automatically. Without it, this program will not work.